### ISSUE TITLE
TiledMap Importer and Processor not working?

### ISSUE THREAD

**Issue by RamiMorrar** [delightedcat/TiledCS.MonoGame#2]
I tried using the example program but it seems that the tiled map Processor and importers aren't working properly? I'm not getting any Tiled Map examples rendering in the game window. 

![Capture](https://user-images.githubusercontent.com/54463476/120753404-fbf70680-c4bf-11eb-8d87-627c27a98a3d.JPG)
![Capture2](https://user-images.githubusercontent.com/54463476/120753532-2779f100-c4c0-11eb-92c9-2c9069d82e0c.JPG)

**Comment by delightedcat** [comment]
Thank you for opening an issue! I was about to try to reproduce the issue, but I'm facing some problems with getting the tool running on my end. I'll get back to you on this later today, most likely.

In the meantime, did you try to compile the `TiledCS.MonoGame.Pipeline` project before trying to compile the example game?

There are no instructions in the README I've noticed. This is due to the fact that I've been focusing on another library for the same purpose, namely [TiledCS.Extensions.MonoGame](https://github.com/delightedcat/TiledCS.Extensions.MonoGame). I'll most likely merge this repository with the other project sometime soon.

**Comment by RamiMorrar** [comment]
> Thank you for opening an issue! I was about to try to reproduce the issue, but I'm facing some problems with getting the tool running on my end. I'll get back to you on this later today, most likely.
> 
> In the meantime, did you try to compile the `TiledCS.MonoGame.Pipeline` project before trying to compile the example game?
> 
> There are no instructions in the README I've noticed. This is due to the fact that I've been focusing on another library for the same purpose, namely [TiledCS.Extensions.MonoGame](https://github.com/delightedcat/TiledCS.Extensions.MonoGame). I'll most likely merge this repository with the other project sometime soon.

I tried compiling the pipeline but no dice. I forgot what the exact error was

**Comment by delightedcat** [comment]
@RamiMorrar Could you try to get the exact error message and paste it here? It's going to be useful for me to solve the issue.

You might also be able to change the reference path in the `Content.mgcb` file in case it doesn't work. I think you also need to change the content importer manually in the tool where it says `Invalid / Missing Importer` after doing so. There is no processor for the custom pipeline seeing as it's not necessary.

**Comment by RamiMorrar** [comment]
I'm getting this error right now. Also it isn't really showing an example on how to draw it. 

![Capture](https://user-images.githubusercontent.com/54463476/120828874-e3174100-c511-11eb-9958-115cd635feaf.JPG)

And are you talking about having it set to copy instead of build in the pipeline window?

**Comment by delightedcat** [comment]
I think the error that you're seeing is caused by having the wrong startup project set. You need the `Desktop` project to be your startup project, because a library on its own cannot be executed.

The example is just some basic code to render a map using the TiledCS library. It gives you an idea on how to implement the library on their own, while the other library I mentioned is more of a library that people can use in their own project directly.

**Comment by RamiMorrar** [comment]
Ah ok. 

But is there a way to draw the tiled in or no?

**Comment by delightedcat** [comment]
Definitely, this library exists as an example to help people with getting started using TiledCS with MonoGame.

Once everything is up and running, you will be presented with a basic window which renders the map provided in the example project's `Content` directory. 

I'm yet to reproduce the error, although I have some problems to fix with my own build of the editor tool.

**Comment by RamiMorrar** [comment]
Alright. I'll still be on the lookout for updates here when you have them. Right now, a lot of things like Extended and Nez dlls are screwed up in the 3.8 GUI pipeline

**Comment by delightedcat** [comment]
Good news, I found a workaround to solve the problem!

It seems like the TiledCS assembly was missing for the content pipeline, so I just included it in [my last commit](https://github.com/delightedcat/TiledCS.MonoGame/commit/1bdeac66439ae38dc3b5cf6016df57821065044c). That said, it does not immediately solve the problem.

To get it working, clone the repository, and open a terminal (CMD or PowerShell on Windows) and run the following command:
```
dotnet build TiledCS.MonoGame.Pipeline
```
Next, temporarily remove the following line from `TiledCS.MonoGame.Desktop.csproj`:
```
<PackageReference Include="MonoGame.Content.Builder.Task" Version="3.8.0.1641" />
```
Now run `dotnet build`. The missing assembly should now be compiled. Add back the line you previously removed from `TiledCS.MonoGame.Desktop.csproj` and run the same command again.

I hope it solves the problem. Feel free to close the issue if you're ready! I should definitely add this to the README, but since I'll be migrating this repository, I'll add it to the new location.

**Comment by RamiMorrar** [comment]
Ok I'll check it out when I get home.
